tv_apps/fling-player/js/fling_player.js

```diff
@@ -206,8 +206,12 @@
       if (!this.isControlPanelHiding()) {
         this._keyNav.pause();
         this._controlPanel.classList.add('fade-out');
+        setTimeout(() => {
+          if (this.isControlPanelHiding()) {
+            this._keyNav.focusOn(this._playButton);
+          }
+        }, 60000);
```

Comment:
1. What would happen if user is controlling the panel after 60 secs?
=> The control panel will show when user is controlling. So it won't affect user's controlling. 
2. Can you use a constant for the value rather than just write it here?
=> Updated